.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,80 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] Implement required functionality to support new platform
+
+- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
+    - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
+    - [ ] Add platform to the Media struct in release/release.go
+    - [ ] Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go
+    - [ ] (Cloud Only) Cloud Images need to have a Images field
+
+- [ ] [fedora-coreos-tracker](https://github.com/coreos/fedora-coreos-tracker/)
+    - [Example PR](https://github.com/coreos/fedora-coreos-tracker/pull/1213/)
+    - [ ] Update the metadata for the new platform
+
+- [ ] (Cloud Only) [stream-metadata-rust](https://github.com/coreos/stream-metadata-rust/)
+    - [Example PR](https://github.com/coreos/stream-metadata-rust/pull/16)
+
+- [ ] [build pipeline](https://github.com/coreos/fedora-coreos-pipeline)
+    - [Example PR](https://github.com/coreos/fedora-coreos-pipeline/pull/500)
+    - [ ] Add platform to the list found in jobs/build.Jenkinsfile for building the new artifact
+
+- [ ] [fedora-coreos-browser](https://github.com/coreos/fedora-coreos-browser)
+    - [Example PR](https://github.com/coreos/fedora-coreos-browser/pull/35)
+    - [ ] Add a list element for new platform in browser/index.html
+
+- [ ] [fedora-coreos-docs](https://github.com/coreos/fedora-coreos-docs)
+    - [Example PR](https://github.com/coreos/fedora-coreos-docs/pull/377)
+    - [ ] Add a provisioning-[platform].adoc that walks through how to setup the new platform
+    - [ ] Add an entry in the modules/ROOT/nav.adoc that points to new documentation
+
+- [ ] [fedora-web](https://pagure.io/fedora-web/websites)
+    - [Example PR](https://pagure.io/fedora-web/websites/pull-request/221#request_diff)
+    - [ ] Add platform to the sites/static/js/coreos-download.js
+
+## At Release
+
+1. Merge changes for:
+    - [ ] coreos-assembler
```

Comment:
Ah okay, I think I see what you mean; its been a while since doing these steps so its getting a bit blurry for me. 

I went ahead and made to separate steps '1','2' both merging separate groups of code, as described above. 

Thank you for your feedback, and I hope the change I made satisfied your expectation.